README.md

```diff
@@ -133,8 +133,7 @@ Now it is time to create a solidity smart contract.
     }
    ```
 
-   - And that's it! your code
-     like [this](https://github.com/BlockDevsUnited/BasicFrontEndTutorial/blob/master/contracts/mood.sol)
+   - And that's it! Your code should look like [this]. (https://github.com/BlockDevsUnited/BasicFrontEndTutorial/blob/master/contracts/mood.sol)
```

Comment:
Can you remove the full-stop here?
[Text](HTTP Link) is markdown syntax for clickable links. The full stop will mess with that syntax.